docs/encyclopedia/temporal-nexus-security.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: nexus-security
+title: Security in Nexus
+sidebar_label: Security
+description: Explore the security aspects of Temporal Nexus, including managing Nexus Endpoints, runtime access control, secure routing, and payload encryption to ensure safe and reliable cross-namespace operations.
+toc_max_heading_level: 4
+slug: /temporalnexus/security
+keywords:
+  - Nexus security
+  - access control
+  - encryption
+tags:
+  - security
+  - temporal
+  - distributed systems
+---
+
+### Security {#security}
+
+#### Managing Nexus Endpoints in the Nexus API Registry
+
+Like `Namespaces`, a Nexus `Endpoint` is an account-scoped resource that is global within your Temporal Cloud account.
```

Comment:
Again, we should explain this in the context of an open source deployment and Cloud.